docs/usage/cards/light.md

```diff
@@ -2,9 +2,52 @@
 title: Lights
 ---
 
+## Light card
 
-Test including external file..
+![Image title](../../assets/img/card_light_combi.png)
+
+This documentation refers to the new "All in one" light card.
+This card merges the following one :
+- legacy ```card_light``` (the old one)
+- legacy ```card_light_slider```
+- legacy ```card_light_slider_collapse```
+- legacy ```card_light_slider_horizontal```
+- custom ```card_light_color``` by basbruss 
+- custom ```card_light_horizontal_color``` by basbruss 
+- custom ```card_light_slider_color``` by basbruss 
+- custom ```card_light_slider_collapse_color``` by basbruss 
+
+!!! warning
+
+    This card has backward compatibilty with older template except custom names and icons. It means variables like `ulm_card_XXX_name` and `ulm_card_XXX_icon` must be replaced by `ulm_card_light_name` and `ulm_card_light_icon`.
```

Comment:
you must remove this indentation.
Perhaps you would like to use here also best a warning admonition?
https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Reply:
nevermind, it is actual a warning admonition that you are using :-D
So ignore that and I will take care on the markdownlinting adjustments later on